packages/eccodes/fxtr_parallel_import.patch

```diff
@@ -0,0 +1,1704 @@
+@@ -889,10 +889,10 @@ static int _read_any(reader *r, int grib_ok, int bufr_ok, int hdf5_ok, int wrap_
```

Comment:
uhmm the patch developed by jmb should contain only a couple fo lines. Are u sure this is the patch or we are comparing same releases?

Reply:
The patch developed by Jean-Marie is in there I checked !